.github/workflows/ci.yml

```
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: [12.x, 14.x, 15.x]
+        architecture: [x64]
+        include:
+          - os: windows-2016
+            node_version: 12
+            architecture: x86
+      fail-fast: false
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set node version to ${{ matrix.node }}
+        uses: actions/setup-node@v1
```

v1 doesn't have arch support. I introduced it in v2
```suggestion
        uses: actions/setup-node@v2
```